Content.Server/Flash/RevoFlashSystem.cs

```diff
@@ -0,0 +1,61 @@
+using Content.Server.Players;
+using Content.Server.Mind.Components;
+using Content.Server.Traitor;
+using Content.Server.Flash.Components;
+using Content.Shared.Roles;
+using Robust.Server.Player;
+using Robust.Shared.Audio;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+namespace Content.Server.Flash
+{
+    internal sealed class RevoHeadFlashSystem : EntitySystem
```

Comment:
```suggestion
    public sealed class RevoHeadFlashSystem : EntitySystem
```

Also this code should be stored with revolutionaries, not flashes.